packages/edit-post/src/components/header/index.js

```diff
@@ -81,9 +89,29 @@ function Header( { onToggleInserter, isInserterOpen } ) {
 					/>
 				) }
 				<PreviewOptions
-					forceIsAutosaveable={ hasActiveMetaboxes }
-					forcePreviewLink={ isSaving ? null : undefined }
-				/>
+					isEnabled={ isPostSaveable }
+					className="edit-post-post-preview-dropdown"
+					deviceType={ deviceType }
+					setDeviceType={ setPreviewDeviceType }
+				>
+					<MenuGroup>
+						<div className="edit-post-header-preview__grouping-external">
+							<PostPreviewButton
```

Comment:
Nit: Potentially, we could extract this to a small component somewhere ini edit-post to avoid having to add selectors... specific to this component here.

Reply:
I think this would make sense for the whole `PreviewOptions` section here too. 🤔

Reply:
Refactored in 2376bfd662f1f3265b8416d137d7581bf610f0a2.